lib/data_layer.ex

```diff
@@ -861,6 +866,146 @@ defmodule AshPostgres.DataLayer do
   defp no_table?(%{from: %{source: {"", _}}}), do: true
   defp no_table?(_), do: false
 
+  # Load bypass aggregates for each result record by querying across all tenants
+  defp load_bypass_aggregates(results, [], _resource, _original_query), do: {:ok, results}
```

Comment:
🤔 I don't think we can realistically support this unfortunately. I think our only real option here is to forbid this behavior when using context multi tenancy and only allow it when using attribute multi tenancy. 

Looking at the code has highlighted to me that there will be a whole host of issues with this kind of approach.

For your needs, are you using attribute or context multitenancy? If it's just attribute multitenancy, it may be as straightforward as ensuring that when building the queries for aggregates not to add the tenant. That may even already work? 

If so, then I think the best bet is to forbid this behavior in Ash core, so no changes end up required here.

Apologies for suggesting this approach to you and then saying it can't be done 😢